### PR TITLE
Add SQS for batching

### PR DESCRIPTION
**Pull request by elseagle**

### CODE REVIEW

File: openstates/settings.py [hunk 1]
```diff
@@ -3,6 +3,9 @@
 
 # settings for realtime flag
 S3_REALTIME_BASE = os.environ.get("S3_REALTIME_BASE")  # e.g 's3://realtime-bucket'
```
Comment (johnseekins) [inline openstates/settings.py:5]: Should we keep the S3 bucket definition if we'll be writing to SQS instead?

File: openstates/scrape/base.py [hunk 2]
```diff
@@ -182,10 +190,11 @@ def save_object(self, obj):
 
             if self.realtime:
 
+                self.output_file_path = str(file_path_)
+
                 s3 = S3FileSystem(anon=False)
 
-                S3_FULL_PATH = settings.S3_REALTIME_BASE + str(file_path_)
-                self.s3_output_path = S3_FULL_PATH
+                S3_FULL_PATH = settings.S3_REALTIME_BASE + self.output_file_path
```
Comment (johnseekins) [inline openstates/scrape/base.py:197]: This string isn't created with an f-string. Just a nit, but perhaps we fix that?

**Review by bfossen-ce — COMMENTED** [review]
Agreed on the couple comments John made but besides that LGTM

File: openstates/settings.py (hunk 1 above)
Comment (elseagle) [reply openstates/settings.py:5]: Yes for now, we are persisting the actuals files for a while. The actual data isn't sent to  SQS yet, just the path to the file on s3. 

cc: @johnseekins @bfossen-ce

File: openstates/scrape/base.py (hunk 2 above)
Comment (elseagle) [reply openstates/scrape/base.py:197]: Yeah, would fix this thanks. Still in draft,  will update

File: openstates/scrape/base.py [hunk 3]
```diff
@@ -122,6 +126,36 @@ def __init__(
...
+        )
+
+        # Send message to SQS queue
+        response = sqs.send_message(
+            QueueUrl=queue_url,
+            DelaySeconds=10,
+            MessageAttributes={
+                "Title": {"DataType": "String", "StringValue": "S3 Output Path"},
+                "Author": {"DataType": "String", "StringValue": "Open States"},
+                "WeeksOn": {"DataType": "Number", "StringValue": "6"},
```
Comment (johnseekins) [inline openstates/scrape/base.py:153]: Can we explain this number? `6` isn't very clear on its own.

Comment (elseagle) [reply openstates/scrape/base.py:153]: > Can we explain this number? `6` isn't very clear on its own.

We don't actually need it, got this from the official doc but it's a sample from the docs, I thought it was important initially

**Review by bfossen-ce — APPROVED** [review]
LGTM

**Pull request merged**